python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -9039,3 +9041,30 @@ def test_pearson_corr_multiindex_dataframe():
     expected = gdf.to_pandas().groupby(level="a").corr("pearson")
 
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        {"a": [np.nan, 1, 2], "b": [None, None, None]},
+        {"a": [1, 2, np.nan, 2], "b": [np.nan, np.nan, np.nan, np.nan]},
```

Comment:
Could we add a testcase of both `NA` and `np.nan` in the same column?

Reply:
Done 👍